config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -131,22 +131,55 @@ resources_for_kubernetes () {
     case $kubernetes in master|$experimental_k8s_version|release-*) cat <<EOF
 resources:
   requests:
-    # these are both a bit below peak usage during build
-    # this is mostly for building kubernetes
-    memory: "9000Mi"
-    # during the tests more like 3-20m is used
-    cpu: 2000m
+    memory: "12Gi"
+    cpu: 6
+  limits:
+    memory: "12Gi"
+    cpu: 6
 EOF
                             ;;
                             *) cat <<EOF
 resources:
   requests:
-    cpu: 2000m
+    memory: "9Gi"
+    cpu: 4
+  limits:
+    memory: "9Gi"
+    cpu: 4
 EOF
                             ;;
     esac | sed -e "s/^/${indentation}/"
 }
 
+job_cluster() {
+  local repo=$1
+
+  # Declare an associative array
+  declare -A cluster_mapping
+
+  cluster_mapping["eks-prow-build-cluster"]="external-snapshotter external-resizer"
```

Comment:
why only snapshotter and resizer repos?

Reply:
I suppose to migrate in steps?

Reply:
Exactly, I loosely plan on doing three to four follow on PRs to migrate all the repos.